website/docs/Examples/EMGProjects/EMG_Joystick.md

```diff
@@ -0,0 +1,67 @@
+---
+id: EMG_Joystick
+title: EMG Joystick
+---
+This tutorial will show you how to control a joystick using EMG data with the OpenBCI GUI. You can then use this muscle activated joystick for any purpose of your choosing! We have developed an tuneable adaptive EMG classifier for accurate control of the joystick. We used this to control a drone as seen in the 2023 OpenBCI TED Talk.
+
+Check out an example video of this joystick being put into action!
+
+<!-- [![EMG Joystick Tutorial](../../assets/TutorialImages/)]() -->
+
+## Materials Required
+
+1.  OpenBCI [Cyton Board](https://shop.openbci.com/collections/frontpage/products/cyton-biosensing-board-8-channel?variant=38958638542)
+2.  [Skintact sticky electrodes](https://shop.openbci.com/collections/frontpage/products/skintact-f301-pediatric-foam-solid-gel-electrodes-30-pack?variant=29467659395) or [IDUN Dryode](https://shop.openbci.com/collections/frontpage/products/idun-dryode-kit)
+3.  [EMG/ECG Snap Electrode Cables](https://shop.openbci.com/collections/frontpage/products/emg-ecg-snap-electrode-cables?variant=32372786958)
+4.  Computer with downloaded [OpenBCI GUI](Software/OpenBCISoftware/01-OpenBCI_GUI.md)
+
+## Step 1: Hardware Assembly
+
+Follow the [EMG Getting Started tutorial](GettingStarted/Biosensing-Setups/02-EMG-Setup.md) to connect the electrodes to your body and the Cyton board, and read data from it using the OpenBCI GUI. For this tutorial, you will need to connect up to four channels. The number of channels depends on how many degrees of freedom needed. Four channels will enable control of both positive and negative X/Y directions of the joystick. Only one ground is needed.
+
+Here is a mapping of channel numbers to X/Y joystick directions: 
+<!-- **Channel 1 = Negative X (left), Channel 2 = Positive X (right), Channel 3 = Positive Y (up), Channel 4 = Negative Y (down)** -->
+
+| Channel | Direction              |
+| ------- | ---------------------- |
+| 1       | - X (left)      |
+| 2       | + X (right)     |
+| 3       | + Y (up)        |
+| 4       | - Y (down)      |
+
+**Important**: The most complicated part of this tutorial can be finding the right placement for each set of electrodes. It is important to find the muscle locations that are the easiest for the user to precisely control. You may need to play around with different locations before finding your optimal electrode setup.
+
+## Step 2: GUI Setup
+
+Start by streaming data using the GUI. Follow this [getting started tutorial](../../Software/OpenBCISoftware/01-OpenBCI_GUI.md/#using-the-openbci-gui) to get the board streaming. You will need to open the EMG Joystick and EMG Settings widgets. 
+
+### EMG Settings
+
+This widget contains the tuneable parameters used for the threshold algorithm that classifies how active each channel is. These parameters should be adjusted to get optimal classification accuracy. The output value is a normalzed value mapped between the lower threshold and upper threshold (between 0-1)
```

Comment:
- classifies -> determines
- to get optimal classification accuracy -> to get the best results
- normalzed -> normalized
- Missing punctuation last sentence.
- Revised last sentence: The output value is a normalized value, from 0 to 1,  mapped between the lower threshold and upper threshold.